index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title: home
+---
+### A weekly review club for Bitcoin Core PRs
+
+<span class="question">What is this?</span> A weekly
+club for reviewing Bitcoin Core PRs in the Freenode `#bitcoin-core-pr-reviews`
+channel at **18:00 UTC on Wednesdays**.
+
+<span class="question">What's it for?</span> To
+help newer contributors learn about the Bitcoin Core review process. Review Club
+is *not* primarily intended to help open PRs get merged (although that might be
+a nice side-effect). The Review Club will give participants the tools and
+knowledge they need in order to take part in the Bitcoin Core review process on
+Github.
+
+<span class="question">Who should take part?</span>
+Anyone who wants to learn about contributing to Bitcoin Core. All are welcome
+to come and ask questions!
+
+<span class="question">What's the benefit for participants?</span> Reviewing
+and testing PRs is the best way to start contributing to Bitcoin Core, but it's
+difficult to know where to start. There are hundreds of open PRs, many require
+a lot of contextual knowledge, and contributors and reviewers often use
+unfamiliar terminology. The Review Club exists to help new contributors start
+contributing!
+
+<span class="question">How do I take part?</span> To
+take part, you should:
+
+- Clone the repo, checkout and build the PR branch and run all tests.
+- Review the code changes and read the comments on the PR.
+- Make a note of any questions you want to ask.
+- Join the IRC channel at **18:00 UTC on wednesday**.
+
+<span class="question">Who runs this?</span> jnewbery
```

Comment:
s/jnewbery/John Newbery/? Using your IRC/GH nick here might seem odd.

Reply:
I prefer to use my nick here than full name. I've added a link to my github profile in case it's not obvious who this is referring to.